lib/cisco_node_utils/vlan.rb

```diff
@@ -60,6 +61,45 @@ def cli_error_check(result)
       fail result[2]['body'] unless result[2]['body'].empty?
     end
 
+    def fabricpath_feature
+      FabricpathGlobal.fabricpath_feature
+    end
+
+    def fabricpath_feature_set(fabricpath_set)
+      FabricpathGlobal.fabricpath_feature_set(fabricpath_set)
+    end
+
+    def mode
+      result = config_get('vlan', 'mode', @vlan_id)
+      return default_mode if result.nil?
+      case result
+      when /fabricpath/i
+        return 'fabricpath'
+      when /ce/i
+        return 'ce'
+      end
```

Comment:
We shouldn't need this regex's directly in the code if we can handle it in the yaml. (I'm harping on the 'show run' again)